Source/WebCore/crypto/cocoa/CryptoAlgorithmAESKWMac.mm

```diff
@@ -42,6 +43,17 @@ static ExceptionOr<Vector<uint8_t>> wrapKeyAESKW(const Vector<uint8_t>& key, con
     return WTFMove(result);
 }
 
+static ExceptionOr<Vector<uint8_t>> wrapKeyAESKWCryptoKit(const Vector<uint8_t>& key, const Vector<uint8_t>& data)
+{
+    // CryptoKit has no way to return the size so assuming 2x input size.
+    Vector<uint8_t> result(2 * data.size());
+    uint64_t resultSize = result.size();
+    if (WebCryptoErrorCodesSuccess != [WebCryptoAesKw wrap:key.data() keySize:key.size() data:data.data() dataSize:data.size() cipherText:result.data() cipherTextSize:&resultSize])
```

Comment:
parameters being used for output isn't great.  Is there a way we can return something like Expected<size_t, ExceptionCode>?

Reply:
We could do that and return a struct instead of an Int where the struct has two values.

Reply:
done.